packages/api/.env

```diff
@@ -1,4 +1,4 @@
-LP_ORCHESTRATORS=[{"address":"https://127.0.0.1:3086","cliAddress":"http://127.0.0.1:3076"}]
+#LP_ORCHESTRATORS=[{"address":"https://127.0.0.1:3086","cliAddress":"http://127.0.0.1:3076","score":1}]
```

Comment:
Does this need to be commented out? If so, WDYT of moving it to the `.env.local.example` file instead?